src/riskMetrics/DynamicRisk.m

```diff
@@ -0,0 +1,72 @@
+function DynamicRisk = DynamicRisk(data)
+%DynamicRisk function that compute the dynamic risk based on current
+%glucose concentration and its rate-of-change, using hyperbolic tangent with
+%pre-defined parameters as amplification function
```

Comment:
Parameters that are fixed should be declared in the comment. 

It's probably better to project this function letting the fixed parameter to be optionally set. See https://github.com/gcappon/agata/blob/master/src/utils/glucoseVectorToTimetable.m for an example on how to deal with optional parameters in AGATA.